### DISCUSSION TITLE
无法安装依赖curl

### DISCUSSION THREAD

**Discussion by DDXDB** [vernesong/OpenClash#2554] [General]
系统为官方OpenWrt 22.03.0-rc5
架构 | Qualcomm Atheros QCA956X ver 1 rev 0
目标平台 | ath79/nand

无法安装依赖curl
curl需要依赖libwolfssl5.4.0.ee39414e显示不可用
官方库仅存在libwolfssl5.3.0.ee39414e

**Comment by vernesong** [comment]
源码编译问题无法提供帮助
